projects/addon-editor/extensions/group/group.extension.ts

```diff
@@ -73,6 +99,31 @@ export const createGroupExtension = (
 
                         return commands.insertContent(result);
                     },
+                setGroupHilite:
+                    (color: string) =>
+                    ({state, editor, dispatch}) => {
```

Comment:
I'm not sure I understand this function 🤔 Could you clarify it for me?

Reply:
update comments